{"sections":[],"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["occ"],"tokens":[{"kind":"keyword","text":"extern"},{"kind":"text","text":" "},{"kind":"typeIdentifier","preciseIdentifier":"c:@T@NSErrorUserInfoKey","text":"NSErrorUserInfoKey"},{"kind":"text","text":" "},{"kind":"keyword","text":"const"},{"kind":"text","text":" "},{"kind":"identifier","text":"SMIErrorNetworkHTTPApiResultErrorCode"},{"kind":"text","text":";"}],"platforms":["iOS"]}]}],"metadata":{"title":"SMIErrorNetworkHTTPApiResultErrorCode","fragments":[{"kind":"identifier","text":"SMIErrorNetworkHTTPApiResultErrorCode"}],"roleHeading":"Global Variable","role":"symbol","modules":[{"name":"SMIClientCore"}],"navigatorTitle":[{"kind":"identifier","text":"SMIErrorNetworkHTTPApiResultErrorCode"}],"symbolKind":"var","externalID":"c:@SMIErrorNetworkHTTPApiResultErrorCode"},"hierarchy":{"paths":[["doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore"]]},"abstract":[{"type":"text","text":"A network API Result error."}],"kind":"symbol","schemaVersion":{"patch":0,"major":0,"minor":3},"identifier":{"interfaceLanguage":"occ","url":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIErrorNetworkHTTPApiResultErrorCode"},"variants":[{"traits":[{"interfaceLanguage":"occ"}],"paths":["\/documentation\/smiclientcore\/smierrornetworkhttpapiresulterrorcode"]}],"references":{"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore":{"url":"\/documentation\/smiclientcore","kind":"symbol","title":"SMIClientCore","abstract":[],"role":"collection","type":"topic","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore"},"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore/SMIErrorNetworkHTTPApiResultErrorCode":{"type":"topic","title":"SMIErrorNetworkHTTPApiResultErrorCode","role":"symbol","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIErrorNetworkHTTPApiResultErrorCode","kind":"symbol","fragments":[{"text":"SMIErrorNetworkHTTPApiResultErrorCode","kind":"identifier"}],"abstract":[{"text":"A network API Result error.","type":"text"}],"url":"\/documentation\/smiclientcore\/smierrornetworkhttpapiresulterrorcode"}}}